ipatests/test_integration/test_acme.py

```diff
@@ -252,3 +264,72 @@ def test_disable_acme_service(self):
             ['curl', '--fail', self.acme_server],
             ok_returncode=22,
         )
+
+    def test_centralize_acme_disable(self):
+        """Test if ACME disable on replica if disabled on master"""
+        result = self.replicas[0].run_command(['ipa-acme-manage', 'status'])
+        assert 'disabled' in result.stdout_text
+
+
+@pytest.mark.skipif(not cainstance.minimum_acme_support(),
+                    reason="does not provide ACME")
+class TestACMECALess(IntegrationTest):
+    """Test to check the CA less replica setup"""
+    num_replicas = 1
+    num_clients = 1
+
+    #########################################
+    #                TODO                   #
+    # refactor install after PR#5147 merged #
+    #########################################
+    @classmethod
+    def install(cls, mh):
+        # cache the acme service uri
+        acme_host = f'{IPA_CA_RECORD}.{cls.master.domain.name}'
+        cls.acme_server = f'https://{acme_host}/acme/directory'
+
+        # install packages before client install in case of IPA DNS problems
+        if not skip_certbot_tests:
+            cls.clients[0].run_command(['dnf', 'install', '-y', 'certbot'])
+        if not skip_mod_md_tests:
+            cls.clients[0].run_command(['dnf', 'install', '-y', 'mod_md'])
```

Comment:
AFAICT this test does not perform certbot or mod_md tests.  We can delete these installation steps and also set
`num_clients = 0` and `num_replicas = 0`.

Reply:
removed. But `num_replicas` remains 1 as it is needed for test.

Reply:
@mrizwan93 yes thank you, you are correct!  Cheers.